docs/Architecture Decision Record/009-git-workflow.md

```diff
@@ -0,0 +1,94 @@
+# 1. Record architecture decisions
+
+Date: 2021-02-23
+
+## Status
+
+Pending
+
+## Context
+
+In order to maintain the principal of Least Privilege, it was decided at the onset of this project that the vendor (Raft) would work from a fork of the government repo and issue pull requests to the government repo from the fork. The vendor would not have write access to the government repository or the government's CircleCI account.
+
+Throughout the project all vendor development work has been done in the vendor's forked repository, while pull requests from the government (OFA and 18F) and even some documentation pull requests from the vendor were made directly to the government repository.
```

Comment:
throughout this documentation, we recommend replacing references to "OFA/HHS + 18F" with the Government.

Reply:
This is done